cmd/fetcher/app/server.go

```diff
@@ -115,6 +115,8 @@ func Run(logger *zap.Logger) {
 	mux.HandleFunc("/specialize", f.SpecializeHandler)
 	mux.HandleFunc("/upload", f.UploadHandler)
 	mux.HandleFunc("/version", f.VersionHandler)
+	mux.HandleFunc("/websocket", f.WebsocketHandler)
+	mux.HandleFunc("/inactive", f.InactiveHandler)
```

Comment:
Similar to above, we could name this endpoint `/endEvent` and event has message `wsConnStopedEvent`. On second thought how about just having a generic `/event` API endpoint and the API internally makes a distinction about which events etc.?

Reply:
We could make this a single POST endpoint but right now it's a very simple get call being made from `Environment` to these  two different endpoints .  Since we're exposing these endpoints to developers who might want to customize their environment , it's nice to have that simplicity IMO. Otherwise, the boilerplate to make a POST request gets added in the environment code. Taking a step back , `event` makes perfect sense to have in endpoint name but I am wondering if that's something a user has to know about? They might start to question the event mechanism in place and retry sending events which probably won't be necessary. WDYT? @vishal-biyani